static/docs/get-started/import-data.md

```diff
@@ -0,0 +1,70 @@
+# Import Data
+
+We've seen how to [push](/doc/get-started/store-data) and
+[pull](/doc/get-started/retrieve-data) data a <abbr>DVC project</abbr>'s
+[remote](/doc/commands-reference/remote). But what if we wanted to integrate a
+dataset or ML model produced in one project into another project? (As an
+[external dependency](/doc/user-guide/external-dependencies))
+
+One way is to download the data (with `wget` or `dvc get`, for example) and use
+`dvc add` to track it locally, but the connection between projects would be
+lost: we couldn't tell where the data came from or whether there are new
+versions available. We offer the `dvc import` command instead (for projects
+hosted on a Git repository). It downloads the data, and creates a
```

Comment:
I think we can omit that it's hosted on Git here

Reply:
I guess "hosted" is not the clearest term, changed to "for projects in a Git repository" to match the (new) [`dvc import` command reference](https://dvc-org-pr-631.herokuapp.com/doc/commands-reference/import). But I think it's an important note to keep, it only takes 6 words.